### PR TITLE
Navigation

### PR DESCRIPTION
**Pull request by DmitryGolubko**
Deployed on https://budget-planning-app.herokuapp.com
- New welcome page with only 2 buttons
image
- added navigation buttons
- skip user confirmation mail send when logging through social networks
- added bootstrap

### CODE REVIEW

File: app/controllers/application_controller.rb [hunk 1]
```diff
@@ -1,5 +1,7 @@
 class ApplicationController < ActionController::Base
   protect_from_forgery with: :exception
-  
-  before_action :authenticate_user!
+  protected
```
Comment (roman-dubrovsky) [inline app/controllers/application_controller.rb]: let's add empty lines between `protected`

File: public/packs/manifest.json [hunk 2]
```diff
@@ -1,8 +1,8 @@
 {
-  "application.js": "/packs/application-855cab8672554126d09a.js",
-  "application.js.map": "/packs/application-855cab8672554126d09a.js.map",
-  "bootstrap.css": "/packs/bootstrap-123f5df71554dc5f28208b4292283f62.css",
-  "bootstrap.css.map": "/packs/bootstrap-123f5df71554dc5f28208b4292283f62.css.map",
-  "bootstrap.js": "/packs/bootstrap-a0ce8e4785e70e78cae5.js",
-  "bootstrap.js.map": "/packs/bootstrap-a0ce8e4785e70e78cae5.js.map"
+  "application.js": "/packs/application-bcf6121e9db4fd2dc7c7.js",
```
Comment (roman-dubrovsky) [inline public/packs/manifest.json]: can you add `manifest.json`  to gitignore?

File: app/controllers/application_controller.rb (hunk 1 above)
Comment (DmitryGolubko) [reply app/controllers/application_controller.rb]: Done!

File: public/packs/manifest.json (hunk 2 above)
Comment (DmitryGolubko) [reply public/packs/manifest.json]: Done :+1:

**Pull request merged**